config/services.xml

```diff
@@ -32,6 +32,7 @@
         <service id="Magento\MagentoCloud\Step\Deploy\InstallUpdate\Install" autowire="false" />
         <service id="Magento\MagentoCloud\Step\Deploy\InstallUpdate\Update" autowire="false" />
         <service id="Magento\MagentoCloud\Step\Deploy\PreDeploy" autowire="false" />
+        <service id="Magento\MagentoCloud\Step\DummyStep" autowire="false" />
```

Comment:
This should be removed

Reply:
No, we need some Step which will be logging about the skipped steps.
Maybe we need to change the name of this class.

Reply:
Please revert step remove functionality.
It must use `remove="true"` attribute instead

Reply:
Functionality doesn't need to be reverted it will be updated If we will decide to use attribute instead.

Reply:
Ok, the class was renamed.